webnn.json

```diff
@@ -415,7 +531,15 @@
           {"name": "userdata", "type": "void", "annotation": "*"},
           {"name": "outputs", "type": "named outputs", "optional": true}
         ]
+      },
+      {
+        "name": "computeSync",
```

Comment:
It looks like it introduces the experimental sync build and compute API. I guess it is for wasm lib investigation according to https://github.com/webmachinelearning/webnn/issues/156#issuecomment-846828170. It probably needs to be mentioned in the commit log and PR message.

Please note the PR for sync build and compute is under review: https://github.com/webmachinelearning/webnn/pull/174